extendedmodlog/eventmixin.py

```diff
@@ -1448,8 +1445,10 @@ async def on_voice_state_update(
             m_id=member.id,
         )
         embed.set_author(
-            name=_("{member} ({m_id}) Voice State Update").format(member=member, m_id=member.id)
+            name=_("{member}").format(member=member),
```

Comment:
This is alsoinconsistent with all the other logging such as member join logging it has the member ID after the members diuscord name.